src/index.html

```diff
@@ -1,17 +1,142 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      class="form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="registrationForm"
+    >
+      <fieldset class="form-section personalInformation">
+        <legend class="title-form-section">Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text" name="surname" autocomplete="off">
+        </label>
+        <label class="form-field">
+          Name:
+          <input type="text" name="name" autocomplete="off">
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input 
+            type="number" 
+            name="how-old" 
+            min="1" 
+            max="100" 
+            value="12"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="dateOfBirth" >
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox" name="acceptTheTerm" >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-section registration">
+        <legend class="title-form-section">Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            name="mail" 
+            placeholder="email@example.com"
+            multiple
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input 
+            type="password" 
+            name="password"
+            minlength="10"
+            maxlength="30"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-section interestingFacts">
+        <legend class="title-form-section">An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label for="loveCats">             
```

Comment:
Is I click on label 'Yes', input radio 'Yes' not selected, please fix
![image](https://user-images.githubusercontent.com/54936044/221536684-6930f030-2bb9-43f1-96a7-12b5b5919f14.png)